quickwit/quickwit-janitor/src/garbage_collection.rs

```diff
@@ -108,7 +108,7 @@ pub async fn run_garbage_collect(
     .await?
     .into_iter()
     // TODO: Update metastore API and push this filter down.
-    .filter(|meta| meta.update_timestamp < grace_period_timestamp)
+    .filter(|meta| meta.update_timestamp <= grace_period_timestamp)
```

Comment:
This makes no difference in practice but allows for running the tests 1 second faster ;)